cdap-ui/app/tracker/controllers/preview-ctrl.js

```diff
@@ -38,19 +39,22 @@ class TrackerPreviewController {
       scope: this.$scope
     };
 
-    let entityType;
     if (this.$state.params.entityType === 'streams') {
-      entityType = 'stream';
+      this.entityType = 'stream';
     } else if (this.$state.params.entityType === 'datasets') {
-      entityType = 'dataset';
+      this.entityType = 'dataset';
     }
 
-    let query = 'SELECT * FROM ' + entityType + '_' + this.$state.params.entityId + ' LIMIT 500';
+    let query = 'SELECT * FROM ' + this.entityType + '_' + this.$state.params.entityId + ' LIMIT 500';
```

Comment:
`ExploreTablesStore` needs to be exported via `common/cask-header.js` and then used here via `window` object.